src/restoration-state-handler.ts

```diff
@@ -1,29 +1,22 @@
-import {
-  SearchType,
-  SortDirection,
-  SortParam,
-} from '@internetarchive/search-service';
+import { SearchType, SortDirection } from '@internetarchive/search-service';
 import { getCookie, setCookie } from 'typescript-cookie';
 import {
-  MetadataSortField,
   FacetOption,
   CollectionBrowserContext,
   CollectionDisplayMode,
   SelectedFacets,
   SortField,
   FacetBucket,
   FacetState,
-  URLFieldToSortField,
-  URLSortField,
   getDefaultSelectedFacets,
-  MetadataFieldToURLField,
+  sortOptionFromAPIString,
+  SORT_OPTIONS,
 } from './models';
 import { arrayEquals } from './utils/array-equals';
 
 export interface RestorationState {
   displayMode?: CollectionDisplayMode;
   searchType?: SearchType;
-  sortParam?: SortParam;
```

Comment:
This property was redundant as we're already passing the sort field & direction that are used to derive it.